app/metrics.yaml

```diff
@@ -1249,3 +1249,29 @@ context_menu:
           open_in_new_tab, open_in_private_tab, open_image_in_new_tab,
           save_image, share_link, copy_link, copy_image_location, share_image
         type: string
+
+search_suggestions:
+  suggestion_tapped:
```

Comment:
For iOS I also added this:

```
  search_tapped:
    type: event
    description: The original query is selected.
    bugs:
      - https://github.com/mozilla-mobile/focus-android/issues/5662
    data_reviews:
      - https://github.com/mozilla-mobile/focus-android/pull/?
    data_sensitivity:
      - interaction
    notification_emails:
      - android-probes@mozilla.com
    expires: "2022-11-01"
```

THis covers the case where the user taps the top entry, which is simply the thing they typed and not a suggestion that came back from the search provider.

Reply:
In Focus the first item is still a suggestion. I can compare the value to the actual typed text if this differentiation helps.